src/integration-test/java/io/aiven/connect/jdbc/pg/PgSqlSinkConnectorIT.java

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright 2020 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.connect.jdbc.pg;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.Future;
+
+import org.apache.kafka.clients.producer.RecordMetadata;
+
+import io.aiven.connect.jdbc.JdbcSinkConnector;
+
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class PgSqlSinkConnectorIT extends AbstractPgSqlAwareIT {
+
+    static final Schema VALUE_SCHEMA = new Schema.Parser().parse(
+            "{"
+                    + "\"type\":\"record\","
+                    + "\"name\":\"pg_sql_types\","
+                    + "\"fields\":"
+                    + "["
+                    + "{\"name\":\"id\",\"type\":\"int\"},"
+                    + "{\"name\":\"json_value\", \"type\":\"string\"},"
+                    + "{\"name\":\"jsonb_value\", \"type\":\"string\"}"
+                    + "]}");
+
+    static final String SELECT_QUERY = "SELECT id, json_value, jsonb_value FROM " + TEST_TOPIC_NAME + " ORDER BY id";
+
+    private final ArrayDeque<String> createdConnectors = new ArrayDeque<>();
+
+    private final Random partitionRnd = new Random();
+
+    @BeforeEach
+    void createDbTable() throws Exception {
+        try (final var stm = pgConnection.createStatement()) {
+            final var createTableSql =
+                    String.format(
+                            "CREATE TABLE %s ("
```

Comment:
Stupid question. Should we also add timestamp as one of the fields?? Maybe also offset

Reply:
Nope. The purpose is to show how to use JSON/JSONB in DB for existing tables.